### ISSUE TITLE
On 200% zoom, visible focus indicator is lost on tabbing out of the new header menu

### ISSUE THREAD

**Issue by puja-iaccessible** [adobe/aem-block-collection#8]
**Actual Behaviour**
On 200% zoom screen resolution, the navigation menu is not modal in nature. Pressing tab after the last menu causes the focus to move out of the menu but there is no visible focus indicator on the screen because the focus moves to an item hidden by the displayed menu. 

**Expected Behaviour**
On 200% zoom, the header menu should either behave like a modal and restrict tabbing out of the menu or it should automatically collapse when focus moves out of the menu either through the tab key or in any other way. Visible focus indicator should always be present on the screen. 

**Steps to Reproduce**
1. Open the URL https://main--franklinsidekickrepo--puja-iaccessible.hlx.page/
2. Set the zoom level of the browser to 200%
3. Press enter on the open navigation button
4. Press tab to go to the end of the list item, then press tab once to check if the next item receives the visual focus

**Applicable Guideline:**
https://www.w3.org/TR/WCAG21/#focus-visible

**Why it matters?**
Losing the visible focus indicator is very hard for low-vision users. When focus lands on elements that are not visible on the screen, low-vision users lose their place and canot use the application.

**Screenshot**
![Visual focus in 200% 5](https://user-images.githubusercontent.com/123964084/219406627-6004e948-aa49-4c5c-b9d1-3464b218b3f7.png)
